CONVENTIONS.md

```diff
@@ -286,6 +286,18 @@ end-to-end job run and the component being tuned uses 350m on the same
 run, the request for the component being tuned should be set to
 100m/600m * 350m ~= 58m.
 
+The following PromQL query illustrates how to calculate the adequate CPU request
+for all containers in the `openshift-monitoring` namespace:
```

Comment:
the query mixes the openshift-etcd and openshift-monitoring namespaces.

Reply:
The formula for calculating the adequate request depends on the etcd usage and request. This is why we have both the target namespace (openshift-monitoring in this case) and the reference namespace (openshift-etcd). 

Does this resolve the question, or did you have something else in mind>

Reply:
Ah right I should have read the paragraph above indeed :)

Having said that all monitoring components (except CMO which runs on control plane nodes) should use SDN as the baseline.

Reply:
Ack. The query should be updated to use SDN as the baseline. I also added a note that pods which run on control-plane nods should be evaluated against etcd.